core/flyout.js

```diff
@@ -446,7 +605,13 @@ Blockly.Flyout.prototype.show = function(xmlList) {
  * @private
  */
 Blockly.Flyout.prototype.layoutBlocks_ = function(blocks, gaps, margin) {
-  // Lay out the blocks vertically.
+  // The blocks need to be visible in order to be laid out and measured
+  // correctly, but we don't want the flyout to show up until it's properly
+  // sized.  Opacity is reset at the end of position().
+  this.svgGroup_.style.opacity = 0;
```

Comment:
This may have been a "fix" for an issue with a different root cause.  I'll take it out and see if anything breaks.